sydent/db/sqlitedb.py

```diff
@@ -186,6 +186,15 @@ def _upgradeSchema(self):
             logger.info("v3 -> v4 schema migration complete")
             self._setSchemaVersion(4)
 
+        if curVer < 5:
+            # Fix lookup_hash index for selecting on mxid instead of medium
+            cur = self.db.cursor()
+            cur.execute("DROP INDEX lookup_hash_medium")
```

Comment:
Should probably have an `IF EXISTS` here if only for consistency, but also see below.